### ISSUE TITLE
auto adjusting Issues with Firefox 

### ISSUE THREAD

**Issue by Puspendert** [rpearce/react-expanding-textarea#33]
In Firefox with react version `16.9.0` or greater, the focus while typing is lost. On mobile firefox, `TextArea` is not maintaining the text state.
Chrome is working fine. Here is a screen recording for reference.
[Video on  youtube](https://www.youtube.com/watch?v=fAQdeE6jQqU)

**Comment by rpearce** [comment]
@Puspendert Thank you for filing this issue. I will check it out ASAP

**Comment by Puspendert** [comment]
@rpearce  Could you please test the resolution. At my end it's working fine after the patch.

**Comment by rpearce** [comment]
Unfortunately, removing that line of code will re-introduce this bug: https://github.com/rpearce/react-expanding-textarea/issues/31. If you want to keep trying, by all means do; otherwise, I'll get to addressing this when it's convenient (it's my next priority).

**Comment by Puspendert** [comment]
I tried reproducing the issue #31 on chrome and firefox, but luckily it's working fine on my system. Not any "auto growing height" issue coming up to me. Will wait for your testing and response on this, till the time I would comment that `auto` part.

versions:
MacOS 10.14.6
Firefox 71.0
Chrome 79.0.3945.88

**Comment by rpearce** [comment]
@raunofreiberg do you have any thoughts on his issue?

**Comment by rpearce** [comment]
I'll try to address this issue this week

**Comment by rpearce** [comment]
_Still haven't gotten back to this yet. On my to-do list._

**Comment by rpearce** [comment]
:eyes:

**Comment by rpearce** [comment]
@Puspendert I am _so_ sorry for the wait on this. I misunderstood, and after looking at it again with fresh eyes, your fix looks good.

You need to alter your pull request to point to this repository's `master` branch. Once you do that, I'll merge it, add you to the contributors list, and publish!

**Comment by Puspendert** [comment]
Please check it. It's my first fork that is going to merge with `master` so no experience. I clicked on the **Merge to Master**

**Comment by rpearce** [comment]
@Puspendert You merged this in to your _own_ fork. The [contributing guidelines](https://github.com/rpearce/react-expanding-textarea/blob/master/CONTRIBUTING.md) specify the steps for contributing, and this is standard in open-source.

Try creating another pull request from your branch (_not_ master) to _this_ repository's `master` branch. Here is a resource that might help: https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github

**Comment by rpearce** [comment]
@Puspendert I decided to not wait any longer for your PR to get sorted out, and I went ahead and resolved the issue. You should be able to see the fix in version `2.1.2`. I also added you to the All Contributors list in the README to thank you for your bug report contribution!
<img width="299" alt="image" src="https://user-images.githubusercontent.com/592876/73804496-e4b8e380-47bb-11ea-84a0-0d4a521b9fb7.png">
